docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Contributing to SpECTRE {#contributing_to_spectre}
+
+The following is a set of guidelines for contributing to SpECTRE,
+which is hosted in the [Simulating eXtreme Spacetimes
+Organization](https://github.com/sxs-collabortion) on GitHub.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[What should I know before I get
+started?](#what-should-i-know-before-i-get-started)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the \ref
+code_of_conduct "SpECTRE Code of Conduct". By participating, you are
+expected to uphold this code. Please report unacceptable behavior to
+[conduct@spectre-code.org](mailto:conduct@spectre-code.org).
+
+## What should I know before I get started?
+
+SpECTRE is being developed in support of our collaborative Simulating
+eXtreme Spacetimes (SXS) research program into the multi-messenger
+astrophysics of neutron star mergers, core-collapse supernovae, and
+gamma-ray bursts.  As such, almost all of the contributors to SpECTRE
+are members of SXS institutions, and a large amount of planning the
+design of SpECTRE is done in internal SXS meetings.  If you are a
+member of SXS and wish to get involved, please contact one of the
+project leaders.
+
+In the future, we hope that SpECTRE can be applied to problems across
+discipline boundaries, and that it can be a true community code.  At
+the present time, however, SpECTRE lacks the capability to solve
```

Comment:
Reword more optimistically: "SpECTRE can't yet solve realistic problems, and broad overview documentation is still in an early, incomplete stage."